main/EDMainMod.F90

```diff
@@ -637,12 +662,15 @@ subroutine ed_update_site( currentSite, bc_in, bc_out )
     ! !LOCAL VARIABLES:
     type (ed_patch_type) , pointer :: currentPatch   
     !-----------------------------------------------------------------------
-
-    call canopy_spread(currentSite)
+    if(hlm_use_sp.eq.ifalse)then
```

Comment:
@rosiealice as I read this bit, by turning off the `canopy_spread` code, this locks in the initial value, which is being set by `init_spread_near_bare_ground` to 1.0 in EDTypesMod.F90.  Not sure whether or not the value of spread matters for SP mode, but likely will init and hold it at 0.0 for bigleaf mode.

Reply:
I think that's fine. We should probably make sure we lock in the same value in both cases.  I don't think it matters what the value is for SP mode. 
 
I am setting it to 1.0 in assign_cohort_sp_properties but maybe I should actually use the patch spread and make sure it is invariant instead?

https://github.com/rosiealice/fates/blob/c49dc43e2427918eb4d100b6884859aa4cecee44/biogeochem/EDPhysiologyMod.F90#L1507